js/ui/paygUnlockDialog.js

```diff
@@ -401,9 +402,17 @@ var PaygUnlockDialog = new Lang.Class({
         });
 
         this._successButton.connect('button-press-event', () => {
+            this._onCreditAdded();
+        });
+
+        this._successButton.connect('key-press-event', () => {
+            this._onCreditAdded();
+        });
+    },
+
+    _onCreditAdded: function() {
```

Comment:
Is this the right name for this function? Usually, a name like `onFoo()` is used for a handler for the `foo` event, not for a function that's notifying a `foo` event.

It might make more sense to name this `_onButtonActivated` or similar. You might also want to check if the button emits a signal covering both cases and just connect to that. I think there's actually a signal named `activate` for that purpose but double-check how it works.